soh/src/code/z_actor.c

```diff
@@ -1997,6 +2031,8 @@ s32 func_8002F434(Actor* actor, GlobalContext* globalCtx, s32 getItemId, f32 xzR
     return false;
 }
 
+// TODO: Rename to GiveItemIdFromActorWithFixedRange or similar
```

Comment:
just a note, as of writing, it is still named `func_8002F554` in https://github.com/zeldaret/oot/blob/master/src/code/z_actor.c